src/main/java/com/google/impactdashboard/server/api_utilities/IamBindingRetriever.java

```diff
@@ -76,16 +82,16 @@ public static IamBindingRetriever create() throws Exception {
           return new SimpleImmutableEntry<>(log.getTimestamp(), auditLog);
         }).collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
 
-
     return timeToAuditLogMap.entrySet().stream().map(entry -> {
+      AtomicReference<Long> secondsFromEpoch = new AtomicReference<>();
```

Comment:
I'm not clear on why this is an AtomicReference. 
Is there relevant multithreading going on here?

Reply:
It was just a piece of code that carried over from a refactor of the code